src/biogeophys/CanopyFluxesMod.F90

```diff
@@ -1007,7 +1066,12 @@ subroutine CanopyFluxes(bounds,  num_exposedvegp, filter_exposedvegp,
             ! changed by K.Sakaguchi from here
             ! transfer coefficient over bare soil is changed to a local variable
             ! just for readability of the code (from line 680)
-            csoilb = vkc / (params_inst%a_coef * (z0mg(c) * uaf(p) / 1.5e-5_r8)**params_inst%a_exp)
+            ! RM: Does this need to be updated if Ya08 is used too? Proposed formulation (definitely double-check!)
```

Comment:
Remove this commented out alternative, that we can't maintain.

Reply:
Removed